### PR TITLE
 Change the default network mode to host in run-tnf-container.sh

### PR DESCRIPTION
**Pull request by aabughosh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,9 +167,10 @@ Optional arguments are:
 
 * `-i` gives a name to a custom TNF container image. Supports local images, as well as images from external registries.
 * `-k` gives a path to one or more kubeconfig files to be used by the container to authenticate with the cluster. Paths must be separated by a colon.
-* `-n` gives the network mode of the container. Defaults to `bridge`. See the [docker run --network parameter reference](https://docs.docker.com/engine/reference/run/#network-settings) for more information on how to configure network settings.
+* `-n` gives the network mode of the container. Defaults set to `host`, if the node is in selinux enable then need to set the value to somthing that are not host. See the [docker run --network parameter reference](https://docs.docker.com/engine/reference/run/#network-settings) for more information on how to configure network settings.
```
Comment (jun-rh) [inline README.md:170]: ... 'host', which requires selinux to be disabled. Alternatively, 'bridge' mode can be used with selinux if TNF_CONTAINER_CLIENT is set to "docker" or running the test as root.

**Review by jun-rh — CHANGES_REQUESTED** [review]
Inline

**Review by wying3 — DISMISSED** [review]

**Review by wying3 — CHANGES_REQUESTED** [review]
the line 182 in  run-tnf-container.sh  need to be changed to $CONTAINER_NETWORK_MODE, otherwise print out will be wrong
this is old bug.

File: run-tnf-container.sh [hunk 2]
```diff
@@ -175,9 +176,11 @@ while [[ $1 == -* ]]; do
       -n) if (($# > 1)); then
             export CONTAINER_NETWORK_MODE=$2; shift
           else
-            export CONTAINER_NETWORK_MODE='host'; shift 2
-          fi ;;
-
+            echo "-n requires an argument" 1>&2
+            exit 1
+          fi
+          echo "-n CONTAINER_NETWORK_MODE"
```
[the remark this answers is not in the document]
Comment (aabughosh) [reply run-tnf-container.sh]: done

**Review by jun-rh — APPROVED** [review]

**Review by HamadiSE — APPROVED** [review]

**Pull request merged**